definitions/npm/formik_v0.11.x/test_formik.js

```diff
@@ -1,38 +1,120 @@
 // @flow
+
 import * as React from "react";
-import {
-  Formik,
-  Form,
-  Field,
-  type FieldProps,
-  type FormikActions,
-  type FormikConfig
+import { Formik, Form, Field } from "formik";
+import type {
+  FieldProps,
+  FormikErrors,
+  FormikTouched,
+  FormikActions,
```

Comment:
Not used?

Reply:
Updated to add tests for FormikActions. You end up using FormikActions some in userland when using Formik, so it makes sense to test these.